### PR TITLE
Um039

### PR DESCRIPTION
**Pull request by eu-genia**
Work IDs are in progress of creation, but I have used derived IDs for now

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
minor questions, thank you

File: ES/ESum039.xml [hunk 1]
```diff
@@ -23,197 +23,88 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                        <title>The treatises of Gregory of Nazianzus</title>
-                        <textLang mainLang="gez"/>
-                     </msItem>
-                  </msItem>
-                  <msItem xml:id="ms_i32">
-                     <locus from="160v" to="162v"/>
-                     <title>The canonical answers of Peter of Alexandria</title>
-                     <textLang mainLang="gez"/>
-                  </msItem>
+               <msContents><summary/>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESum039.xml:37]: This is the msContents element with the summary to have the entire summary for the whole MS on top, I always do it (and hope everyone does) for composite manuscripts, where the actual contents is described further for each part separately

File: ES/ESum039.xml [hunk 2]
```diff
@@ -225,15 +116,499 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                              <material key="parchment"/>
+                           </support>
+                           <extent>Made of <measure unit="leaf">38</measure> folia, in <measure unit="quire">5</measure> quires. 
+                              
+                              
+                           </extent>
+                           
+                           <collation>
+                              <list>
+                                 <item xml:id="q1" n="I">
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESum039.xml:192]: @PietroLiuzzo is it a problem to use the Roman numbers? I tried to keep as much as possible from the "source" description

[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESum039.xml]: I have updated the numbers to Arabic

File: ES/ESum039.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply ES/ESum039.xml:37]: (after the physical description of the entire MS there is no msContents, there are separate msContents for each msPart)

**Review by thea-m — APPROVED** [review]

**Pull request merged**